politeiawww/backend.go

```diff
@@ -2108,10 +2110,11 @@ func (b *backend) ProcessLikeComment(lc www.LikeComment, user *database.User) (*
 		return nil, err
 	}
 
+	b.Lock()
```

Comment:
I've done this because of the results of a comment like are now calculated on www. Before it was done on d, so the sequence was `validate like -> request d -> reply` now we have `validate like -> request d -> calculate like results -> reply`. 
Now that we have this calculation at the end of the function, I thought it would make sense to make the whole operation atomic to avoid the following situation:
- Two users voted up on a comment, user 1 see that the voting score is updated accordingly, user 2 see the voting score increased by two (his vote and the user 1 vote).

However, as per our discussion on slack, this race seems pretty benign as it doesn't compromise the data integrity and keeps the results up to date for the client. Therefore, I am reverting this and keeping the lock as it was before.

Reply:
I agree.  The race is benign and preventing it is not worth the performance hit that we would take by locking during the politeiad call.